examples/Session05/maillroom_test.py

```diff
@@ -174,6 +174,16 @@ def input_sc(textfile):
     while bool:
         print(welcome)
         selection = input('Please type a number between 1 & 4:\n')
+<<<<<<< HEAD
```

Comment:
these weird things are "merge conflicts".

But you really shouldn't be changing things in the examples dir anyway. YOu want to copy from tehre to your own dir to mess with stuff.

if you do change things in examples (or solutions, or...) then don't git commit them!

always do a git status before you commit.

and don't do a "git commit -a" (note the -a) if there are changes that are not in your own dir.